### ISSUE TITLE
[5.2.1] fontawsome.css has no font-face to load the font files

### ISSUE THREAD

**Issue by peter1szalatnay** [joomla/joomla-cms#44451]
### Steps to reproduce the issue

Don't know if this is a bug or how it should work. But the fontawesome.css in the vendor folder has no font-face for the font files loaded.

Guess because brands.scss, regular.scss and solid.scss is not compiled into it. 

Is this vendor library only supposed to be compiled into your own library or be able to used as standalone?

### Expected result

Font awesome icons are loaded when using the CSS.

### Actual result

The font files are not loaded.

### System information (as much as possible)

Joomla 5.2.1

**Comment by peter1szalatnay** [comment]
In my component I'm building I want to use the vanilla fontawsome.css from the media/vendor folder. And right now it cannot be used as there is no font files loaded in it.

That is why I am questioning if it is meant to be used as standalone or should be compiled from the scss files.

**Comment by hans2103** [comment]
After installation of Joomla the fonts (ttf, woff2) can be found in media/vendor/font-awesome-free/webfonts.
If they are not present in yours, then there might be an issue with your installation

**Comment by Fedik** [comment]
Yeah, that a bug. default fontawesome.css does not include woff2.
There was a fix for j4 https://github.com/joomla/joomla-cms/pull/31364 , but for j5 it probably need a litle more work.

**Comment by Fedik** [comment]
Note: you can use https://github.com/joomla/joomla-cms/blob/e6518fb5781f5da5edc1a465ff1a48ccbd1aaed8/templates/cassiopeia/joomla.asset.json#L104-L108

In your template, the same as joomla cassiopeia do, which also includes some classes for core `icon-`.

**Comment by pAnd0rASBG** [comment]
I think, the proper way here would be to build what you need based on the supplied scss files:
https://docs.fontawesome.com/web/use-with/scss

**Comment by peter1szalatnay** [comment]
> Note: you can use
> 
> [joomla-cms/templates/cassiopeia/joomla.asset.json](https://github.com/joomla/joomla-cms/blob/e6518fb5781f5da5edc1a465ff1a48ccbd1aaed8/templates/cassiopeia/joomla.asset.json#L104-L108)
> 
> Lines 104 to 108 in [e6518fb](/joomla/joomla-cms/commit/e6518fb5781f5da5edc1a465ff1a48ccbd1aaed8)
> 
>  { 
>    "name": "fontawesome", 
>    "type": "style", 
>    "uri": "system/joomla-fontawesome.min.css" 
>  } 
> In your template, the same as joomla cassiopeia do, which also includes some classes for core `icon-`.

Yes I know, but that still don't change the main issue that the vendor/fontawesome-free is not compiled with font-faces. But if that is by design then it all good.

@Fedik as you pointed out maybe just create a fontawsome-all.scss in the folder that includes all files. This should be compiled on Joomla build, and now we have both fontawsome.css broken and fontawsome-all.css that works.

**Comment by Fedik** [comment]
> maybe just create a fontawsome-all.scss in the folder that includes all files

Yes, kind of that. But I think it is enough to replace fontawsome.css to fontawsome-all.css, beacause fontawsome.css as it is now is useless anyway.
